sqlserver/tests/conftest.py

```diff
@@ -0,0 +1,105 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+import time
+import sys
+
+import pytest
+try:
+    import pyodbc
+except ImportError:
+    pyodbc = None
+
+from datadog_checks.dev import docker_run, get_docker_hostname, RetryError
+
+
+HOST = get_docker_hostname()
+PORT = 1433
+HERE = os.path.dirname(os.path.abspath(__file__))
+
+
+def lib_tds_path():
+    """
+    This is definitely ugly but should do the trick most of the times. On OSX
+    we can point unixODBC directly to the FreeTDS client library. On linux instead
+    we need to define the 'FreeTDS' driver in odbcinst.ini
+    """
+    if sys.platform == 'darwin':
+        return '/usr/local/lib/libtdsodbc.so'
```

Comment:
why is this different than from here https://github.com/DataDog/integrations-core/pull/1926/files#diff-8bf5f5276576f8f979bc13ac314c1c94R5